### PR TITLE
LPS-160574 Automated tests - Testray 2

### PR DESCRIPTION
**Pull request by joelrocha68**
https://issues.liferay.com/browse/LPS-160574

### CODE REVIEW

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:stop

**Comment by AfonsoHSantos** [comment]
LGTM!

**Review by AfonsoHSantos — APPROVED** [review]

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestraySuitesAdministrator.testcase [hunk 1]
```diff
@@ -187,7 +187,7 @@ definition {
 				requirementName = "ATF_Requirements Summary");
 		}
 
-		task ("And given that the user adds a Smart Suite") {
+		task ("And given that the user add a Smart Suite") {
```
Comment (barrosdennys) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestraySuitesAdministrator.testcase:190]: The old form "adds" is the correct one in this case. @joelrocha68 would you mind to change it back to "adds"?

**Comment by joelrocha68** [comment]
ci:forward

**Pull request closed without merging**